catalog-spec/catalog-spec.md

```diff
@@ -30,6 +30,7 @@ also a valid STAC Catalog.
 | Element         | Type          | Description                                                  |
 | --------------- | ------------- | ------------------------------------------------------------ |
 | stac_version    | string        | **REQUIRED.** The STAC version the catalog implements. STAC versions can be mixed, but please keep the [recommended best practices](../best-practices.md#mixing-stac-versions) in mind. |
+| type            | string        | **REQUIRED.** Set to `Catalog` if this catalog only implements the catalog spec, or `Collection` if it additional meets the [collection](../collection-spec/collection-spec.md) requirements and should be treated by clients as such. |
```

Comment:
All of our models in `stac4s` have `Stac`- prefixes so it wouldn't look strange to me if we went that route (re: your [PR comment](https://github.com/radiantearth/stac-spec/pull/971#issuecomment-773821337))

Reply:
Cool, good to know. May be worth including.